bootstrap/api/endpoint_test.go

```diff
@@ -317,24 +340,19 @@ func TestUpdate(t *testing.T) {
 	bs := newBootstrapServer(svc)
 
 	c := bootstrap.Config{
-		ExternalID:  cfg.ExternalID,
-		ExternalKey: cfg.ExternalKey,
-		MFChannels:  cfg.MFChannels,
-		Content:     cfg.Content,
+		ExternalID:  addReq.ExternalID,
+		ExternalKey: addReq.ExternalKey,
+		MFChannels:  []bootstrap.Channel{bootstrap.Channel{ID: "1"}},
```

Comment:
Maybe you can extract this ID to a const

Reply:
What's this ID? Aren't channel IDs in the form of UUIDv4?